extensions-contrib/tdigestsketch/src/main/java/org/apache/druid/query/aggregation/tdigestsketch/TDigestSketchAggregator.java

```diff
@@ -56,6 +56,10 @@ public TDigestSketchAggregator(ColumnValueSelector selector, @Nullable Integer c
   @Override
   public void aggregate()
   {
+    Object obj = selector.getObject();
```

Comment:
nit: can you re-use `obj` below instead of calling `selector.getObject` for each if branch?

Reply:
Thanks for the review, @clintropolis. Pushed a commit to clean it up.